it/test/uk/gov/hmrc/tradergoodsprofilesrouter/GetSingleRecordIntegrationSpec.scala

```diff
@@ -431,7 +431,7 @@ class GetSingleRecordIntegrationSpec
 
   private def sendRequestAndWait(url: String) =
     // TODO: After Drop 1.1 this should be removed and use the request without the X-CLient-ID header -  Ticket: TGP-2014
-    if (appConfig.isDrop1_1_enabled)
+    if (appConfig.isClientIdHeaderDisabled)
```

Comment:
we may do not need to do that as the isClientIdHeaderDisabled will always be false